### ISSUE TITLE
Convert to ES modules

### ISSUE THREAD

**Issue by arikimolloy** [ag-grid/ag-grid-webcomponent#6]
Converting this to an ES module will allow for easy extending of AgGrid. I have done this locally and extended AgGrid to add my own functionality and it works well.

**Comment by socketdigital** [comment]
Yes please drop html imports, Chrome throws a deprecation notice on the simple grid demo:

https://www.ag-grid.com/best-web-component-data-grid/examples/simple-grid/simple-grid-example.html

[Deprecation] HTML Imports is deprecated and will be removed in M80, around February 2020. Please use ES modules instead. See https://www.chromestatus.com/features/5144752345317376 and https://developers.google.com/web/updates/2019/07/web-components-time-to-upgrade for more details.
